aicsimageio/readers/tiff_reader.py

```diff
@@ -276,12 +284,8 @@ def _read_immediate(self) -> xr.DataArray:
                     image_data,
                     dims=dims,
                     coords=coords,
-                    attrs=metadata,
+                    attrs={
+                        constants.METADATA_UNPROCESSED: tiff_tags,
+                        constants.METADATA_PROCESSED: tiff_tags[270].value,
```

Comment:
it would be good to make this a symbolic constant `TIFFTAG_IMAGEDESCRIPTION` or maybe there is one from tifffile.py you can use.

Reply:
The way tifffile handles it would be:

```python
IMAGE_DESCRIPTION_TAG = TIFF.TAGS.get("ImageDescription")
```

I think I may just make a constant that is `IMAGE_DESCRIPTION_TAG = 270` for simplicity?